### ISSUE TITLE
HSL color references don't work

### ISSUE THREAD

**Issue by marcprux** [uwdata/vega#23]
Load the bar.json example in the editor and change:

``` json
          "fill": {"value": "steelblue"},
```

to:

``` json
          "fill": { "h": {"value": 0.2}, "s": {"value": 0.5}, "l": { "value": 0.25 } }
```

And observe that it doesn't render any colors. Vega 1.5's editor renders this without problems.

Is it possible that the d3 code generation isn't being executed the same way?

**Comment by arvind** [comment]
Good catch, I think adding SignalRefs in there screwed something up. The properties (ValueRefs) would be an excellent candidate for thorough unit testing, if you were interested and had the time.

**Comment by marcprux** [comment]
Sure, I'll add in some tests once https://github.com/uwdata/vega/pull/21 https://github.com/uwdata/vega/pull/21 gets merged.

**Comment by arvind** [comment]
Fixed in 4aa2b78. Thanks Marc!
